cp-attendance-hybrid/src/modules/profile/profile.page.tsx

```diff
@@ -61,15 +61,15 @@ export const Profile: React.FC<ProfileScreenProps> = ({ navigation }) => {
           <EditModalItemTitle>{'Nome: '}</EditModalItemTitle>
           <EditModalItemText>{user.user?.username}</EditModalItemText>
         </EditModalItem>
-        <EditModalItem>
+        {/* <EditModalItem>
```

Comment:
Tem algum motivo pra estar comentado e não deletado? Se for o caso deixaria um TODO pra indicar qnd descomentar/deletar/sei lá